lib/article_json/export/apple_news/exporter.rb

```diff
@@ -23,6 +23,8 @@ def components
               ArticleJSON::Export::AppleNews::Elements::Base
                 .build(element)
                 &.export
+            end.reject do |hash|
```

Comment:
How about `end.reject { |hash| hash&.empty? }`

Reply:
Amened

Reply:
We actually have nil values too that we want to remove.

Reply:
Made it a one liner.